scripts/composer/ScriptHandler.php

```diff
@@ -40,11 +41,10 @@ public static function createRequiredFiles(Event $event) {
       $fs->copy($drupalRoot . '/sites/default/default.settings.php', $drupalRoot . '/sites/default/settings.php');
       require_once $drupalRoot . '/core/includes/bootstrap.inc';
       require_once $drupalRoot . '/core/includes/install.inc';
-      $settings['config_directories'] = [
-        CONFIG_SYNC_DIRECTORY => (object) [
-          'value' => Path::makeRelative($drupalFinder->getComposerRoot() . '/config/sync', $drupalRoot),
-          'required' => TRUE,
-        ],
+      new Settings([]);
```

Comment:
This line looks quite strange. New object created but never used afterwards.

Reply:
As I read it, `new Settings([]);` initialises `$settings` array and sets `config_sync_directory` value there afterwards.

Reply:
To test out: 
1. delete `web/sites/default/settings.php`,
2. run `lando composer install`,
3. check that new `web/sites/default/settings.php` is created with `$settings['config_sync_directory'] = '../config/sync';` defined at the end of the file.

Reply:
Some might even ask if we need `settings.php` file creation functionality because we already have `settings.php` in our codebase, but let it be just in case ;)